tracer/src/Datadog.Trace/Vendors/Microsoft.OpenApi/Extensions/OpenApiTypeMapper.cs

```diff
@@ -86,54 +86,5 @@ public static OpenApiSchema MapTypeToOpenApiPrimitiveType(this Type type)
                 ? result()
                 : new() { Type = "string" };
         }
-
-        /// <summary>
-        /// Maps an OpenAPI data type and format to a simple type.
-        /// </summary>
-        /// <param name="schema">The OpenApi data type</param>
-        /// <returns>The simple type</returns>
-        /// <exception cref="ArgumentNullException"></exception>
-        public static Type MapOpenApiPrimitiveTypeToSimpleType(this OpenApiSchema schema)
```

Comment:
The changes in this commit shouldn't be made manually, they should be included in the vendoring tool. The point of the tool is to make sure the vendoring is repeatable, so these changes should be encoded in the tool itself.